data/human/deep missions.txt

```diff
@@ -2431,3 +2431,330 @@ event "battle in arneb end"
+	on complete
+		event "Stone Waiting more" 60
+		payment 25000
+		dialog `Helheim's hustle and bustle threatens to overwhelm the senses, yet Skaldgar, for all his age and illness, seems to have a skip in his step as he saunters out of your ship and into the teeming masses.`
+			`	Just as he is about to be swept up in the current of humanity, Skaldgar turns and gives you one last wave, and then is gone.`
+		
+event "Stone Waiting more"
+
+
+
+mission "Stone of our Fathers 3"
+	name "Stone of our Fathers"
+	description "Take Skaldgar's stone back to his family on <destination>."
+	minor
+	to offer
+		has "event: Stone Waiting more"
+		has "Stone of our Fathers 2: done"
+	source "Helheim"
+	destination "Norn"
+	on offer
+		conversation
+			`As always, Helheim is packed to the brim with transient workers coming and going, with dreams of either getting rich or getting out. A youngish looking man in a dirty roustabout jacket waves at you and approaches.`
+			`	"Yer that Cap'n that Skaldgar always went on about, aren'tcha?" The young man attempts to imitate Skaldgar with a deep and ponderous voice, "Him that knows the currents of space like we know our stone, and the like." The man grins sheepishly, " Ol' Skaldgar always had a way with words, he did."`
+			`	The young man then looks down, somberly, and pulls from his pocket a small, well-worn stone with some strange markings on it and holds it out to you.`
+			`	"He wanted," he begins haltingly, "he wanted you, only you, to take this back to his folk on Norn." He swallows. "It's the old ways, it is. This, it needs to be returned, and not by a stranger." The young man then shrugs. "Best to letcha know, I ain't have no money to pay you. This would have to be done as a favor for the ol' man."`
+			choice
+				`	(You reach out and accept the stone.)`
+					goto accept
+				`	"Nope. Not going to do it. I run a ship, not a charity."`
+			`	The young roustabout nods knowingly. "Guess Skaldgar was wrong about you, but I understand. We all gotta eat." He turns and slides back into the endless crowd.`
+				decline
+			label accept
+			`	You're surprised at how cool the smooth stone feels in your palm.`
+			`	He smiles sadly. "Skaldgar always had a nose for good folks. Anyways, he told me he had a nephew named Helm who worked at the Norn spaceport. Find 'em and he can getcha and this stone where it needs to go." The young man than looks you in the eye and gives a slow respectful nod, "For Ol' Skaldgar, may he be findin' ore in hel."`
+			`	And with that, the man disappears into the crowd.`
+				accept
+	on complete
+		dialog `It turns out there is no need to search for Helm, as he is waiting for you when you land.`
+			`	"I figured for you to be returning, it would either be with Skaldgar or with" —he pauses— " news." He looks around. "And I don't see Skaldgar."`
```

Comment:
I think the intent is "if you're returning, then it would be either with Skaldgar or news," and there's just been words removed to fit the character's speech patterns, in which case the period would change its meaning. Thoughts on keeping it in?
```suggestion
			`	"I figured for you to be returning, it would either be with Skaldgar or with" - he pauses - "news." He looks around. "And I don't see Skaldgar."`
```